### ISSUE TITLE
Icons for stairs direction are not correctly displayed

### ISSUE THREAD

**Issue by jaimeMF** [streetcomplete/StreetComplete#4317]
As shown in the screenshot the icons in the quest asking for the stairs direction are wrongly displayed. I am using version 45.2 from F-Droid, some versions ago (around May or June?) it was correctly displayed. I am running it on an old phone with Android 5.1, I do not have a phone with a more recent version to check it.

![Screenshot_2022-08-20-18-02-23](https://user-images.githubusercontent.com/1239727/186980022-391aa3ed-4547-41df-a3e4-14fae1696668.png)

**Comment by mnalis** [comment]
Yes, it seems to be rendering wrongly...  Those are supposed to be arrows, and on my SC 45.2 from F-Droid they seem to render fine on my Android 10.

@jaimeMF Does the problem persist after phone reboot?
Have you perhaps been changing `Text and display size` in Android settings? 

![small_Screenshot_20220827_004428_de westnordost streetcomplete](https://user-images.githubusercontent.com/156656/186999993-3f87313c-dc92-4ff7-ba4c-52bce0a31f46.jpg)

**Comment by jaimeMF** [comment]
> Does the problem persist after phone reboot?

It is the same after a reboot.

> Have you perhaps been changing `Text and display size` in Android settings?

No.

If it only happens in this old version of Android, I would understand if it is not worth to investigate.

**Comment by riQQ** [comment]
> I am using version 45.2 from F-Droid, some versions ago (around May or June?) it was correctly displayed.

Can you pinpoint the exact version that introduced this display issue?

**Comment by jaimeMF** [comment]
> > I am using version 45.2 from F-Droid, some versions ago (around May or June?) it was correctly displayed.
> 
> Can you pinpoint the exact version that introduced this display issue?

On 44.1 it is shown correctly and in 45.0 it is wrong. But I had not noticed that starting with 45.0 the directions arrows are shown directly below the question, while in previous versions you have to first select the way.

![Screenshot_44_1](https://user-images.githubusercontent.com/1239727/187046427-33e581ab-1e58-4ae2-a12e-30b597af575d.png)

**Comment by westnordost** [comment]
Reproducable with an Android 5.1 emulator.

It looks like Android 5.1 has problems parsing this SVG path correctly:
```m32 10-28 22h16l4 22h16l4-22h16z```
Instead, it requires this to work:
```m32 10l-28 22h16l4 22h16l4-22h16z```

In the end, I would say that this is an issue of the SVG to Vector Drawable tool integrated in Android studio. If Android 5.1 does not support properly the above, the tool should not export it that way. But I very much doubt that Google would be inclined to fix that since Android 5.1 is so old already.

So anyway, I fixed the path data by hand. If you find any more icons that seem to be not complete, let us know and we will have an easy fix for that.

**Comment by jaimeMF** [comment]
Thank you for finding the cause and fixing it.
